nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -213,9 +219,84 @@ public static String serializeSymbol(@Nullable Symbol symbol) {
         return symbol.name.toString();
       case METHOD:
       case CONSTRUCTOR:
-        return symbol.toString();
+        return serializeMethodSignature((Symbol.MethodSymbol) symbol);
       default:
         return symbol.flatName().toString();
     }
   }
+
+  /**
+   * Serializes the signature of the given {@link Symbol.MethodSymbol} to a string.
+   *
+   * @param methodSymbol The method symbol to serialize.
+   * @return The serialized method symbol.
+   */
+  private static String serializeMethodSignature(Symbol.MethodSymbol methodSymbol) {
+    if (methodSymbol == null) {
+      return "null";
+    }
```

Comment:
we don't need this since `methodSymbol` cannot be `null`

Reply:
cf0d63dbd9438e9614a9326eea287bc460a711c5